### PR TITLE
Add basic tests to cover examples in README.md

### PR DESCRIPTION
**Pull request by hannahilea**
In service of #3 (albeit not fully comprehensive). 

Also added GHA and README badges; same comments as https://github.com/SamuraiAku/SPDX.jl/pull/34.

### CODE REVIEW

**Comment by hannahilea** [comment]
@SamuraiAku codecov will need to be enabled here too, as in SPDX.jl

**Comment by SamuraiAku** [comment]
Change the YML file so that it is testing Julia 1.8 and latest, same as in SPDX.jl

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ github.event_name }}
+    # Run on push's or non-draft PRs
+    if: (github.event_name == 'push') || (github.event.pull_request.draft == false)
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - "1.6" # LTS
```
Comment (hannahilea) [inline .github/workflows/CI.yml:24]:
```suggestion
          - "1.8"
```

**Comment by hannahilea** [comment]
updated to 1.8

**Pull request merged**